client/src/App.js

```diff
@@ -1,14 +1,26 @@
-import HomePage from "./components/pages/HomePage.js";
 import GlobalStyle from "./GlobalStyle";
 import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```

Comment:
GlobalStyle and BrowserRouter could be used in the index.js

Reply:
Maybe it could, I think it doesn´t make much difference. In our lessons, we put it in the App.js.